lib/elements/clone.py

```diff
@@ -62,86 +51,87 @@ def clone(self):
            type='float')
     @cache
     def clone_fill_angle(self):
-        return self.get_float_param('angle') or None
+        return self.get_float_param('angle')
 
     @property
     @param('flip_angle',
            _('Flip angle'),
-           tooltip=_("Flip automatically calucalted angle if it appears to be wrong."),
+           tooltip=_("Flip automatically calculated angle if it appears to be wrong."),
            type='boolean')
     @cache
     def flip_angle(self):
-        return self.get_boolean_param('flip_angle')
+        return self.get_boolean_param('flip_angle') or False
```

Comment:
You can just write it like this:

` return self.get_boolean_param('flip_angle', False)`